legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/protocols/pure/vX_X_X/models/metamodel_connection.pure

```diff
@@ -55,6 +55,18 @@ Class meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::a
 {
 }
 
+Class meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::authentication::GCPWorkloadIdentityFederationAuthenticationStrategy extends meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::authentication::AuthenticationStrategy
+{
+    workloadProjectNumber : String[1];
+    serviceAccountEmail : String[1];
+    gcpScope: String[1];
+    workloadPoolId : String[1];
+    workloadProviderId: String[1];
+    discoveryUrl: String[1];
+    clientId: String[1];
```

Comment:
Should this be broken down into 2 separate strategies ?  One that uses AWS as the IDP and one that uses Ping/other as the IDP.

'discoveryUrl'/'clientId' make sense only for IDPs that require them. So why ask users of AWS IDP to provide dummy values for these attributes ?